ide/xml.core/src/org/netbeans/modules/xml/api/model/GrammarQueryManager.java

```diff
@@ -108,7 +108,7 @@ private static class DefaultQueryManager extends GrammarQueryManager {
 
         private static final String FOLDER = "Plugins/XML/GrammarQueryManagers";// NOI18N
         
-        private Lookup.Result registrations;
+        private Lookup.Result<?> registrations;
```

Comment:
```suggestion
        private Lookup.Result<GrammarQueryManager> registrations;
```

After this the `Iterator` returned by `getRegistrations` can get a sane generic signature and then the usesite needs to be updated (in `enabled`).

Reply:
I fixed the declaration. As for the `getRegistrations` , I think I fixed that as best that I could figure out. Let me know if you think I made a mistake or needs more work. It was unclear to me about the `usesite` reference.